src/shadowbox/server/server_access_key.ts

```diff
@@ -313,7 +313,7 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
       if (limitBytes === undefined) {
         limitBytes = Number.POSITIVE_INFINITY;
       }
-      accessKey.isOverDataLimit = usageBytes > limitBytes;
+      accessKey.isOverDataLimit = usageBytes >= limitBytes;
```

Comment:
Thanks for the fix.

I agree that we should disable the key if it reaches the data limit. However, we must rename the variable. Perhaps `reachedDataLimit`?
That's because `isOverDataLimit` is no longer accurate. Being equal is not being over.

Reply:
+1 for `reachedDataLimit`.

As a side note, this `equal` which considers 0 as no access also fixes something else. With this change, now setting the limit to 0 byte will immediately disconnect a connected user.

Last time I checked it's wasn't immediate.